### PR TITLE
Clean up rendered text trees

### PR DESCRIPTION
**Pull request by Mytherin**
This PR cleans up the rendering of text trees in explain/explain analyze in a few ways:

* Remove the `Stringified:` label, instead add a new `__text__` label that shows text without a header
* Move `Extra Cardinality` to `__extra_cardinality__`, and align rendering of extra cardinality across boxes horizontally
* Only use line separators between the operator name and the extra info, use spaces otherwise
* Inline short headers and avoid spaces between short headers

Here's the explain output of TPC-H Q09 with `PRAGMA disabled_optimizers='compressed_materialization'` (which adds a lot of noisy functions): 

#### New Tree
```
┌───────────────────────────┐
│          ORDER_BY         │
│    ────────────────────   │
│     profit.nation ASC     │
│     profit.o_year DESC    │
└─────────────┬─────────────┘
┌─────────────┴─────────────┐
│       HASH_GROUP_BY       │
│    ────────────────────   │
│          Groups:          │
│             #0            │
│             #1            │
│                           │
│    Aggregates: sum(#2)    │
│                           │
│           EC: 25          │
└─────────────┬─────────────┘
┌─────────────┴─────────────┐
│         PROJECTION        │
│    ────────────────────   │
│           nation          │
│           o_year          │
│           amount          │
│                           │
│        EC: 13950920       │
└─────────────┬─────────────┘
┌─────────────┴─────────────┐
│         PROJECTION        │
│    ────────────────────   │
│           nation          │
│    "year"(o_orderdate)    │
│           amount          │
│                           │
│        EC: 13950920       │
└─────────────┬─────────────┘
┌─────────────┴─────────────┐
│         HASH_JOIN         │
│    ────────────────────   │
│      Join Type: INNER     │
│                           │
│        Conditions:        ├──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│  l_orderkey = o_orderkey  │                                                                                                                                  │
│                           │                                                                                                                                  │
│        EC: 13950920       │                                                                                                                                  │
└─────────────┬─────────────┘                                                                                                                                  │
┌─────────────┴─────────────┐                                                                                                                    ┌─────────────┴─────────────┐
│         HASH_JOIN         │                                                                                                                    │         SEQ_SCAN          │
│    ────────────────────   │                                                                                                                    │    ────────────────────   │
│      Join Type: INNER     │                                                                                                                    │           orders          │
│                           │                                                                                                                    │                           │
│        Conditions:        ├──────────────┐                                                                                                     │        Projections:       │
│   l_suppkey = s_suppkey   │              │                                                                                                     │         o_orderkey        │
│   l_partkey = ps_partkey  │              │                                                                                                     │        o_orderdate        │
│                           │              │                                                                                                     │                           │
│        EC: 12708070       │              │                                                                                                     │        EC: 15000000       │
└─────────────┬─────────────┘              │                                                                                                     └───────────────────────────┘
┌─────────────┴─────────────┐┌─────────────┴─────────────┐
│         SEQ_SCAN          ││         HASH_JOIN         │
│    ────────────────────   ││    ────────────────────   │
│          lineitem         ││      Join Type: INNER     │
│                           ││                           │
│        Projections:       ││        Conditions:        │
│         l_suppkey         ││   ps_suppkey = s_suppkey  │
│         l_partkey         ││                           ├───────────────────────────────────────────┐
│         l_orderkey        ││        Build Min: 1       │                                           │
│      l_extendedprice      ││     Build Max: 100000     │                                           │
│         l_discount        ││                           │                                           │
│         l_quantity        ││                           │                                           │
│                           ││                           │                                           │
│        EC: 59986052       ││         EC: 772956        │                                           │
└───────────────────────────┘└─────────────┬─────────────┘                                           │
                             ┌─────────────┴─────────────┐                             ┌─────────────┴─────────────┐
                             │         HASH_JOIN         │                             │         HASH_JOIN         │
                             │    ────────────────────   │                             │    ────────────────────   │
                             │      Join Type: INNER     │                             │      Join Type: INNER     │
                             │                           │                             │                           │
                             │        Conditions:        │                             │        Conditions:        │
                             │   ps_partkey = p_partkey  ├──────────────┐              │ s_nationkey = n_nationkey ├──────────────┐
                             │                           │              │              │                           │              │
                             │                           │              │              │        Build Min: 0       │              │
                             │                           │              │              │       Build Max: 24       │              │
                             │                           │              │              │                           │              │
                             │         EC: 877050        │              │              │         EC: 100000        │              │
                             └─────────────┬─────────────┘              │              └─────────────┬─────────────┘              │
                             ┌─────────────┴─────────────┐┌─────────────┴─────────────┐┌─────────────┴─────────────┐┌─────────────┴─────────────┐
                             │         SEQ_SCAN          ││           FILTER          ││         SEQ_SCAN          ││         SEQ_SCAN          │
                             │    ────────────────────   ││    ────────────────────   ││    ────────────────────   ││    ────────────────────   │
                             │          partsupp         ││ contains(p_name, 'green') ││          supplier         ││           nation          │
                             │                           ││                           ││                           ││                           │
                             │        Projections:       ││                           ││        Projections:       ││        Projections:       │
                             │         ps_suppkey        ││                           ││         s_suppkey         ││        n_nationkey        │
                             │         ps_partkey        ││                           ││        s_nationkey        ││           n_name          │
                             │       ps_supplycost       ││                           ││                           ││                           │
                             │                           ││                           ││                           ││                           │
                             │        EC: 8000000        ││         EC: 400000        ││         EC: 100000        ││           EC: 25          │
                             └───────────────────────────┘└─────────────┬─────────────┘└───────────────────────────┘└───────────────────────────┘
                                                          ┌─────────────┴─────────────┐
                                                          │         SEQ_SCAN          │
                                                          │    ────────────────────   │
                                                          │            part           │
                                                          │                           │
                                                          │        Projections:       │
                                                          │         p_partkey         │
                                                          │           p_name          │
                                                          │                           │
                                                          │         EC: 400000        │
                                                          └───────────────────────────┘

```

#### Previous Tree (main)
```
┌───────────────────────────┐
│          ORDER_BY         │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│         Order By:         │
│     profit.nation ASC     │
│     profit.o_year DESC    │
└─────────────┬─────────────┘
┌─────────────┴─────────────┐
│       HASH_GROUP_BY       │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│          Groups:          │
│             #0            │
│             #1            │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│        Aggregates:        │
│          sum(#2)          │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│   Estimated Cardinality:  │
│             25            │
└─────────────┬─────────────┘
┌─────────────┴─────────────┐
│         PROJECTION        │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│        Projections:       │
│           nation          │
│           o_year          │
│           amount          │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│   Estimated Cardinality:  │
│          13950920         │
└─────────────┬─────────────┘
┌─────────────┴─────────────┐
│         PROJECTION        │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│        Projections:       │
│           nation          │
│    "year"(o_orderdate)    │
│           amount          │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│   Estimated Cardinality:  │
│          13950920         │
└─────────────┬─────────────┘
┌─────────────┴─────────────┐
│         HASH_JOIN         │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│         Join Type:        │
│           INNER           │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│        Conditions:        ├──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│  l_orderkey = o_orderkey  │                                                                                                                                  │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                                                                                                  │
│   Estimated Cardinality:  │                                                                                                                                  │
│          13950920         │                                                                                                                                  │
└─────────────┬─────────────┘                                                                                                                                  │
┌─────────────┴─────────────┐                                                                                                                    ┌─────────────┴─────────────┐
│         HASH_JOIN         │                                                                                                                    │         SEQ_SCAN          │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                                                                                    │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│         Join Type:        │                                                                                                                    │        Stringified:       │
│           INNER           │                                                                                                                    │           orders          │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                                                                                    │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│        Conditions:        ├──────────────┐                                                                                                     │        Projections:       │
│   l_suppkey = s_suppkey   │              │                                                                                                     │         o_orderkey        │
│   l_partkey = ps_partkey  │              │                                                                                                     │        o_orderdate        │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │              │                                                                                                     │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│   Estimated Cardinality:  │              │                                                                                                     │   Estimated Cardinality:  │
│          12708070         │              │                                                                                                     │          15000000         │
└─────────────┬─────────────┘              │                                                                                                     └───────────────────────────┘
┌─────────────┴─────────────┐┌─────────────┴─────────────┐
│         SEQ_SCAN          ││         HASH_JOIN         │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│        Stringified:       ││         Join Type:        │
│          lineitem         ││           INNER           │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│        Projections:       ││        Conditions:        │
│         l_suppkey         ││   ps_suppkey = s_suppkey  │
│         l_partkey         ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│         l_orderkey        ││         Build Min:        ├───────────────────────────────────────────┐
│      l_extendedprice      ││             1             │                                           │
│         l_discount        ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                           │
│         l_quantity        ││         Build Max:        │                                           │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││           100000          │                                           │
│   Estimated Cardinality:  ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                           │
│          59986052         ││   Estimated Cardinality:  │                                           │
│                           ││           772956          │                                           │
└───────────────────────────┘└─────────────┬─────────────┘                                           │
                             ┌─────────────┴─────────────┐                             ┌─────────────┴─────────────┐
                             │         HASH_JOIN         │                             │         HASH_JOIN         │
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                             │         Join Type:        │                             │         Join Type:        │
                             │           INNER           │                             │           INNER           │
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                             │        Conditions:        │                             │        Conditions:        │
                             │   ps_partkey = p_partkey  │                             │ s_nationkey = n_nationkey │
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                             │   Estimated Cardinality:  ├──────────────┐              │         Build Min:        ├──────────────┐
                             │           877050          │              │              │             0             │              │
                             │                           │              │              │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │              │
                             │                           │              │              │         Build Max:        │              │
                             │                           │              │              │             24            │              │
                             │                           │              │              │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │              │
                             │                           │              │              │   Estimated Cardinality:  │              │
                             │                           │              │              │           100000          │              │
                             └─────────────┬─────────────┘              │              └─────────────┬─────────────┘              │
                             ┌─────────────┴─────────────┐┌─────────────┴─────────────┐┌─────────────┴─────────────┐┌─────────────┴─────────────┐
                             │         SEQ_SCAN          ││           FILTER          ││         SEQ_SCAN          ││         SEQ_SCAN          │
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                             │        Stringified:       ││           Name:           ││        Stringified:       ││        Stringified:       │
                             │          partsupp         ││ contains(p_name, 'green') ││          supplier         ││           nation          │
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                             │        Projections:       ││   Estimated Cardinality:  ││        Projections:       ││        Projections:       │
                             │         ps_suppkey        ││           400000          ││         s_suppkey         ││        n_nationkey        │
                             │         ps_partkey        ││                           ││        s_nationkey        ││           n_name          │
                             │       ps_supplycost       ││                           ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││                           ││   Estimated Cardinality:  ││   Estimated Cardinality:  │
                             │   Estimated Cardinality:  ││                           ││           100000          ││             25            │
                             │          8000000          ││                           ││                           ││                           │
                             └───────────────────────────┘└─────────────┬─────────────┘└───────────────────────────┘└───────────────────────────┘
                                                          ┌─────────────┴─────────────┐
                                                          │         SEQ_SCAN          │
                                                          │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                                                          │        Stringified:       │
                                                          │            part           │
                                                          │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                                                          │        Projections:       │
                                                          │         p_partkey         │
                                                          │           p_name          │
                                                          │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                                                          │   Estimated Cardinality:  │
                                                          │           400000          │
                                                          └───────────────────────────┘

```

#### v1.0

```
┌───────────────────────────┐                                                                                                                                                 
│          ORDER_BY         │                                                                                                                                                 
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                                                                                                                 
│          ORDERS:          │                                                                                                                                                 
│     profit.nation ASC     │                                                                                                                                                 
│     profit.o_year DESC    │                                                                                                                                                 
└─────────────┬─────────────┘                                                                                                                                                                              
┌─────────────┴─────────────┐                                                                                                                                                 
│       HASH_GROUP_BY       │                                                                                                                                                 
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                                                                                                                 
│             #0            │                                                                                                                                                 
│             #1            │                                                                                                                                                 
│          sum(#2)          │                                                                                                                                                 
└─────────────┬─────────────┘                                                                                                                                                                              
┌─────────────┴─────────────┐                                                                                                                                                 
│         PROJECTION        │                                                                                                                                                 
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                                                                                                                 
│           nation          │                                                                                                                                                 
│           o_year          │                                                                                                                                                 
│           amount          │                                                                                                                                                 
└─────────────┬─────────────┘                                                                                                                                                                              
┌─────────────┴─────────────┐                                                                                                                                                 
│         PROJECTION        │                                                                                                                                                 
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                                                                                                                 
│           nation          │                                                                                                                                                 
│     year(o_orderdate)     │                                                                                                                                                 
│           amount          │                                                                                                                                                 
└─────────────┬─────────────┘                                                                                                                                                                              
┌─────────────┴─────────────┐                                                                                                                                                 
│         HASH_JOIN         │                                                                                                                                                 
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                                                                                                                 
│           INNER           │                                                                                                                                                 
│  o_orderkey = l_orderkey  ├──────────────┐                                                                                                                                  
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │              │                                                                                                                                  
│        EC: 13950920       │              │                                                                                                                                  
└─────────────┬─────────────┘              │                                                                                                                                                               
┌─────────────┴─────────────┐┌─────────────┴─────────────┐                                                                                                                    
│         SEQ_SCAN          ││         HASH_JOIN         │                                                                                                                    
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                                                                                    
│           orders          ││           INNER           │                                                                                                                    
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   l_suppkey = ps_suppkey  │                                                                                                                    
│         o_orderkey        ││   l_partkey = ps_partkey  ├──────────────┐                                                                                                     
│        o_orderdate        ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │              │                                                                                                     
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││        EC: 12708070       │              │                                                                                                     
│        EC: 15000000       ││                           │              │                                                                                                     
└───────────────────────────┘└─────────────┬─────────────┘              │                                                                                                                                  
                             ┌─────────────┴─────────────┐┌─────────────┴─────────────┐                                                                                       
                             │         SEQ_SCAN          ││         HASH_JOIN         │                                                                                       
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                                                       
                             │          lineitem         ││           INNER           │                                                                                       
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ps_suppkey = s_suppkey  │                                                                                       
                             │         l_suppkey         ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                                                       
                             │         l_partkey         ││        Build Min: 1       │                                                                                       
                             │         l_orderkey        ││     Build Max: 100000     ├───────────────────────────────────────────┐                                           
                             │      l_extendedprice      ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                           │                                           
                             │         l_discount        ││         EC: 772956        │                                           │                                           
                             │         l_quantity        ││                           │                                           │                                           
                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││                           │                                           │                                           
                             │        EC: 59986052       ││                           │                                           │                                           
                             └───────────────────────────┘└─────────────┬─────────────┘                                           │                                                                        
                                                          ┌─────────────┴─────────────┐                             ┌─────────────┴─────────────┐                             
                                                          │         HASH_JOIN         │                             │         HASH_JOIN         │                             
                                                          │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                             │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                             
                                                          │           INNER           │                             │           INNER           │                             
                                                          │   ps_partkey = p_partkey  │                             │ s_nationkey = n_nationkey │                             
                                                          │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ├──────────────┐              │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ├──────────────┐              
                                                          │         EC: 877050        │              │              │        Build Min: 0       │              │              
                                                          │                           │              │              │       Build Max: 24       │              │              
                                                          │                           │              │              │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │              │              
                                                          │                           │              │              │         EC: 100000        │              │              
                                                          └─────────────┬─────────────┘              │              └─────────────┬─────────────┘              │                                           
                                                          ┌─────────────┴─────────────┐┌─────────────┴─────────────┐┌─────────────┴─────────────┐┌─────────────┴─────────────┐
                                                          │         SEQ_SCAN          ││           FILTER          ││         SEQ_SCAN          ││         SEQ_SCAN          │
                                                          │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                                                          │          partsupp         ││ contains(p_name, 'green') ││          supplier         ││           nation          │
                                                          │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                                                          │         ps_suppkey        ││         EC: 400000        ││         s_suppkey         ││        n_nationkey        │
                                                          │         ps_partkey        ││                           ││        s_nationkey        ││           n_name          │
                                                          │       ps_supplycost       ││                           ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
                                                          │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   ││                           ││         EC: 100000        ││           EC: 25          │
                                                          │        EC: 8000000        ││                           ││                           ││                           │
                                                          └───────────────────────────┘└─────────────┬─────────────┘└───────────────────────────┘└───────────────────────────┘                             
                                                                                       ┌─────────────┴─────────────┐                                                          
                                                                                       │         SEQ_SCAN          │                                                          
                                                                                       │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                          
                                                                                       │            part           │                                                          
                                                                                       │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                          
                                                                                       │         p_partkey         │                                                          
                                                                                       │           p_name          │                                                          
                                                                                       │   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │                                                          
                                                                                       │         EC: 400000        │                                                          
                                                                                       └───────────────────────────┘                                                                                       

```

### CODE REVIEW

File: src/execution/operator/filter/physical_filter.cpp [hunk 1]
```diff
@@ -54,8 +54,8 @@ OperatorResultType PhysicalFilter::ExecuteInternal(ExecutionContext &context, Da
 
 InsertionOrderPreservingMap<string> PhysicalFilter::ParamsToString() const {
 	InsertionOrderPreservingMap<string> result;
-	result["Name"] = expression->GetName();
-	result["Estimated Cardinality"] = StringUtil::Format("%llu", estimated_cardinality);
+	result["__text__"] = expression->GetName();
```
Comment (Tishj) [inline src/execution/operator/filter/physical_filter.cpp:57]: I get that the excessive use of headers causes a lot of noise, but the initial reason for that is for the JSONTreeRenderer, which needs a key->value. I'm afraid that `__text__` is not clear enough, can you fix up the `json_tree_renderer.cpp` as well?

`html` and `graphviz` also need a bit of work if the source `extra_text` changes like this

Comment (Mytherin) [reply src/execution/operator/filter/physical_filter.cpp]: I've changed it now so the tree renderer has an option `UsesRawKeyNames()` - if that is not set we convert the underscore names back into title case (so `__estimated_cardinality__` becomes `Estimated Cardinality`). The only difference for the other parsers should now be that `Stringified` has become `Text`.

File: src/common/render_tree.cpp [hunk 2]
```diff
@@ -204,6 +204,26 @@ unique_ptr<RenderTree> RenderTree::CreateRenderTree(const ProfilingNode &op) {
 	return CreateTree<ProfilingNode>(op);
 }
 
+void RenderTree::SanitizeKeyNames() {
```
Comment (Tishj) [inline src/common/render_tree.cpp:207]: The QueryProfiler handles the JSON version of the EXPLAIN ANALYZE output, which also needs this I think

```sql
statement ok
PRAGMA enable_verification

statement ok
CREATE TABLE integers AS SELECT * FROM range(100) tbl(i);

statement error
EXPLAIN (FORMAT JSON, ANALYZE) SELECT SUM(i) FROM (SELECT * FROM integers i1, integers i2 UNION ALL SELECT * FROM integers i1, integers i2);
----
```

```
explain_key     explain_value
VARCHAR VARCHAR
[ Rows: 1]
analyzed_plan   {
    "query": "",
    "cpu_time": 0.0,
    "extra_info": {},
    "cumulative_cardinality": 0,
    "operator_cardinality": 0,
    "cumulative_rows_scanned": 0,
    "operator_rows_scanned": 0,
    "operator_timing": 0.0,
    "timings": [
        {
            "annotation": "optimizer",
            "timing": 0.001717
        },
        {
            "annotation": "optimizer > build_side_probe_side",
            "timing": 0.000072
        },
        {
            "annotation": "optimizer > column_lifetime",
            "timing": 0.000019999999999999998
        },
        {
            "annotation": "optimizer > common_aggregate",
            "timing": 0.000012
        },
        {
            "annotation": "optimizer > common_subexpressions",
            "timing": 0.000014
        },
        {
            "annotation": "optimizer > cte_filter_pusher",
            "timing": 0.000002
        },
        {
            "annotation": "optimizer > deliminator",
            "timing": 0.000008
        },
        {
            "annotation": "optimizer > duplicate_groups",
            "timing": 0.000013
        },
        {
            "annotation": "optimizer > expression_rewriter",
            "timing": 0.00003
        },
        {
            "annotation": "optimizer > filter_pullup",
            "timing": 0.000003
        },
        {
            "annotation": "optimizer > filter_pushdown",
            "timing": 0.000047
        },
        {
            "annotation": "optimizer > in_clause",
            "timing": 0.000008
        },
        {
            "annotation": "optimizer > join_filter_pushdown",
            "timing": 0.000004
        },
        {
            "annotation": "optimizer > join_order",
            "timing": 0.000374
        },
        {
            "annotation": "optimizer > limit_pushdown",
            "timing": 0.000002
        },
        {
            "annotation": "optimizer > regex_range",
            "timing": 0.000002
        },
        {
            "annotation": "optimizer > reorder_filter",
            "timing": 0.000013
        },
        {
            "annotation": "optimizer > statistics_propagation",
            "timing": 0.000085
        },
        {
            "annotation": "optimizer > top_n",
            "timing": 0.000002
        },
        {
            "annotation": "optimizer > unnest_rewriter",
            "timing": 0.000003
        },
        {
            "annotation": "optimizer > unused_columns",
            "timing": 0.000041
        },
        {
            "annotation": "physical_planner",
            "timing": 0.000232
        },
        {
            "annotation": "physical_planner > column_binding",
            "timing": 0.000015
        },
        {
            "annotation": "physical_planner > create_plan",
            "timing": 0.00017
        },
        {
            "annotation": "physical_planner > resolve_types",
            "timing": 0.00001
        },
        {
            "annotation": "planner",
            "timing": 0.0007549999999999999
        },
        {
            "annotation": "planner > binder",
            "timing": 0.000744
        }
    ],
    "children": [
        {
            "name": "EXPLAIN_ANALYZE",
            "operator_timing": 0.000001,
            "operator_rows_scanned": 0,
            "cumulative_rows_scanned": 0,
            "operator_cardinality": 0,
            "cumulative_cardinality": 0,
            "extra_info": {},
            "cpu_time": 0.0,
            "children": [
                {
                    "name": "UNGROUPED_AGGREGATE",
                    "cpu_time": 0.0,
                    "extra_info": {
                        "Aggregates": "sum_no_overflow(#0)"
                    },
                    "cumulative_cardinality": 0,
                    "operator_cardinality": 1,
                    "cumulative_rows_scanned": 0,
                    "operator_rows_scanned": 0,
                    "operator_timing": 0.0003589999999999995,
                    "children": [
                        {
                            "name": "PROJECTION",
                            "operator_timing": 0.004364000000000003,
                            "operator_rows_scanned": 0,
                            "cumulative_rows_scanned": 0,
                            "operator_cardinality": 20000,
                            "cumulative_cardinality": 0,
                            "extra_info": {
                                "__projections__": "i",
                                "__estimated_cardinality__": "20000"
                            },
                            "cpu_time": 0.0,
                            "children": [
                                {
                                    "name": "UNION",
                                    "cpu_time": 0.0,
                                    "extra_info": {},
                                    "cumulative_cardinality": 0,
                                    "operator_cardinality": 0,
                                    "cumulative_rows_scanned": 0,
                                    "operator_rows_scanned": 0,
                                    "operator_timing": 0.0,
                                    "children": [
                                        {
                                            "name": "PROJECTION",
                                            "operator_timing": 0.002186000000000003,
                                            "operator_rows_scanned": 0,
                                            "cumulative_rows_scanned": 0,
                                            "operator_cardinality": 10000,
                                            "cumulative_cardinality": 0,
                                            "extra_info": {
                                                "__projections__": "i",
                                                "__estimated_cardinality__": "10000"
                                            },
                                            "cpu_time": 0.0,
                                            "children": [
                                                {
                                                    "name": "CROSS_PRODUCT",
                                                    "cpu_time": 0.0,
                                                    "extra_info": {},
                                                    "cumulative_cardinality": 0,
                                                    "operator_cardinality": 10000,
                                                    "cumulative_rows_scanned": 0,
                                                    "operator_rows_scanned": 0,
                                                    "operator_timing": 0.00022499999999999964,
                                                    "children": [
                                                        {
                                                            "name": "SEQ_SCAN ",
                                                            "operator_timing": 0.0000049999999999999996,
                                                            "operator_rows_scanned": 100,
                                                            "cumulative_rows_scanned": 0,
                                                            "operator_cardinality": 100,
                                                            "cumulative_cardinality": 0,
                                                            "extra_info": {
                                                                "__text__": "integers",
                                                                "Projections": "",
                                                                "__estimated_cardinality__": "100"
                                                            },
                                                            "cpu_time": 0.0,
                                                            "children": []
                                                        },
                                                        {
                                                            "name": "SEQ_SCAN ",
                                                            "operator_timing": 0.000051,
                                                            "operator_rows_scanned": 100,
                                                            "cumulative_rows_scanned": 0,
                                                            "operator_cardinality": 100,
                                                            "cumulative_cardinality": 0,
                                                            "extra_info": {
                                                                "__text__": "integers",
                                                                "Projections": "i",
                                                                "__estimated_cardinality__": "100"
                                                            },
                                                            "cpu_time": 0.0,
                                                            "children": []
                                                        }
                                                    ]
                                                }
                                            ]
                                        },
                                        {
                                            "name": "PROJECTION",
                                            "operator_timing": 0.002168000000000002,
                                            "operator_rows_scanned": 0,
                                            "cumulative_rows_scanned": 0,
                                            "operator_cardinality": 10000,
                                            "cumulative_cardinality": 0,
                                            "extra_info": {
                                                "__projections__": "i",
                                                "__estimated_cardinality__": "10000"
                                            },
                                            "cpu_time": 0.0,
                                            "children": [
                                                {
                                                    "name": "CROSS_PRODUCT",
                                                    "cpu_time": 0.0,
                                                    "extra_info": {},
                                                    "cumulative_cardinality": 0,
                                                    "operator_cardinality": 10000,
                                                    "cumulative_rows_scanned": 0,
                                                    "operator_rows_scanned": 0,
                                                    "operator_timing": 0.0002469999999999996,
                                                    "children": [
                                                        {
                                                            "name": "SEQ_SCAN ",
                                                            "operator_timing": 0.000014000000000000001,
                                                            "operator_rows_scanned": 100,
                                                            "cumulative_rows_scanned": 0,
                                                            "operator_cardinality": 100,
                                                            "cumulative_cardinality": 0,
                                                            "extra_info": {
                                                                "__text__": "integers",
                                                                "Projections": "",
                                                                "__estimated_cardinality__": "100"
                                                            },
                                                            "cpu_time": 0.0,
                                                            "children": []
                                                        },
                                                        {
                                                            "name": "SEQ_SCAN ",
                                                            "operator_timing": 0.00005,
                                                            "operator_rows_scanned": 100,
                                                            "cumulative_rows_scanned": 0,
                                                            "operator_cardinality": 100,
                                                            "cumulative_cardinality": 0,
                                                            "extra_info": {
                                                                "__text__": "integers",
                                                                "Projections": "i",
                                                                "__estimated_cardinality__": "100"
                                                            },
                                                            "cpu_time": 0.0,
                                                            "children": []
                                                        }
                                                    ]
                                                }
                                            ]
                                        }
                                    ]
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ]
}
```

Comment (Mytherin) [reply src/common/render_tree.cpp:207]: Good point - I've done the sanitization there as well

**Comment by Tishj** [comment]
I see you're having fun with the context added by the key->value map change 🚀 Glad to see it was useful and enables some QoL improvements to the explain (analyze) output!

**Comment by Mytherin** [comment]
> I see you're having fun with the context added by the key->value map change 🚀 Glad to see it was useful and enables some QoL improvements to the explain (analyze) output!

Yep, definitely very useful :)

**Pull request merged**